packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -51,17 +56,29 @@ export function PublishArticlePanel({
             })}
           />
         )}
+
         <DateTimePicker
-          dateTime={publishDate}
+          dateTime={publishedAt}
           label={t('articleEditor.panels.publishDate')}
-          changeDate={date => setPublishDate(date)}
+          changeDate={date => setPublishedAt(date)}
         />
         <DateTimePicker
-          dateTime={updateDate}
+          dateTime={updatedAt}
           label={t('articleEditor.panels.updateDate')}
-          changeDate={date => setUpdateDate(date)}
+          changeDate={date => setupdatedAt(date)}
         />
 
+        <Panel
+          header={t('articleEditor.panels.advancedOptions')}
+          collapsible
+          className="availableFromPublishPanel">
+          <DateTimePicker
```

Comment:
i tested this locally and always had a date set for publishAt. I think it's because the `<DateTimePicker />` component automatically sets a date if the input is `undefined`.